scripts/reboot

```diff
@@ -198,6 +198,21 @@ function parse_options()
     done
 }
 
+function linecard_reboot_notify_supervisor()
+{
+    is_linecard=$(python3 -c 'from sonic_py_common import device_info; print("True") if device_info.is_chassis() == True and device_info.is_supervisor() == False else print("False")')
+    if [ $is_linecard == "True" ]; then
+        key=$(sonic-db-cli STATE_DB keys "CHASSIS_MODULE_TABLE|LINE-CARD*")
+        status=$?
+        if [ $status -eq 0 ]; then
+            module="${key#CHASSIS_MODULE_TABLE}"
+            if [ ! -z module ]; then
+                sonic-db-cli CHASSIS_STATE_DB hset "CHASSIS_MODULE_REBOOT_INFO_TABLE${module}" "reboot" "expected"
```

Comment:
can we check return code and print an error if this does not succeed for any reason.

Reply:
@mlok-nokia , yes. lets add a syslog error message for this case... That will be helpful for debugging purposes.

Reply:
Fixed. Add log as suggestion to address this comment.